tests/Resources/Page/ODH/JupyterHub/HighAvailability.robot

```diff
@@ -100,6 +100,7 @@ Wait Until JH Deployment Is Ready
 Rollout JupyterHub
     [Documentation]     Rollouts JupyterHub deployment and wait until it is finished
     Start Rollout   dc_name=jupyterhub  namespace=redhat-ods-applications
+    Wait Until Rollout Is Started       namespace=redhat-ods-applications  label_selector=app=jupyterhub
```

Comment:
I think you should pass the `${previous_pods}` as an argument to `Wait Until Rollout Is Started`, and you fetch them before doing `Start Rollout`. In the cases in which the rollout is started by saving changes to the settings the same strategy could be applied, and the previous pods are fetched before saving the changes.
It feels like there is a risk of `Wait until rollout is started` failing because it fetches the new pods when doing the first `OC get` otherwise.

Reply:
hm, yes you're right. I will change it, thanks!

Reply:
done